test/test_gen.py

```diff
@@ -17,6 +19,21 @@ def clean_up():
         rimraf(os.path.join('.', file + 'temp.nc'))
 
 
+def sentinel2_products_exist(path: str) -> bool:
+    """
+    Test if given *path* is likely a Sentinel-3 product path.
+
+    :param path: (directory) path
+    :return: True, if so
+    """
+    if os.path.isdir(path):
+        for fname in os.listdir(path):
+            if fname.endswith('.nc'):
+                return True
+    else:
+        return False
+
+
```

Comment:
This test is likely too weak. There are characteristic files in a Sentinel-3 product. Check if they are present.